ui/components/app/signature-request-original/index.scss

```diff
@@ -193,7 +196,7 @@
 
   &__rows {
     height: 100%;
-    overflow-y: scroll;
+    overflow-y: auto;
```

Comment:
Using auto will hide this empty scrollbar for the message container:
<img width="381" alt="Screenshot 2022-08-17 at 7 45 46 PM" src="https://user-images.githubusercontent.com/20778143/185177888-b6251883-eb0b-4d7d-8022-c43e564faec0.png">